.tekton/retag-collector.yaml

```diff
@@ -38,16 +38,19 @@ spec:
   - name: input-image-tag-makefile-target
     value: collector-tag
 
-  workspaces:
-  - name: git-auth
-    secret:
-      secretName: '{{ git_auth_secret }}'
-
   pipelineRef:
     name: retag-pipeline
 
+  taskRunTemplate:
+    serviceAccountName: build-pipeline-collector
```

Comment:
It's interesting if this would work for these component-less pipelines.

Reply:
It does! The ServiceAccount exists, because there is a component for `collector` (pipeline in the collector repository) that has the necessary credentials.